misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -101,3 +104,30 @@ data class WebUnixDomainSocketConfig(
   /** If true, the listener will support H2C. */
   val h2c: Boolean? = true
 )
+
+data class CorsConfig(
```

Comment:
Defaults as defined here: https://www.eclipse.org/jetty/documentation/current/cross-origin-filter.html

I don't agree with all of them, but I'm possibly just being biased.

Reply:
What changes would you make?!

Reply:
val allowCredentials: Boolean = true
and 
val allowedOrigins: Array<String> = arrayOf("*")

seems a little bit too lenient for a feature that effectively allows opening up a security policy

Reply:
(Going with defaults for now)